plugins/meta/route-override/README.md

```diff
@@ -0,0 +1,64 @@
+# route-override: Meta CNI plugin for overriding IP route
+
+## Overview
+ route-override IPAM works as meta CNI plugin to override IP route given by previous CNI plugins.
+It is useful in a case with [network-attachment-definition](https://github.com/K8sNetworkPlumbingWG/multi-net-spec).
```

Comment:
What case?

Reply:
When a container has multiple interface, user (i.e. container process) want to send the traffic to specific destination (i.e. VPN traffic goes to `tun0`, not `eth0`). In such case, we need to modify routing table of the network namespace and the plugin does. Clarified?

Reply:
I added more description in README